internal/wsmessage.go

```diff
@@ -23,9 +23,9 @@ func DecodeWSMessage(bytes []byte, msg proto.Message) error {
 		}
 		// Skip the header. It really is just a single zero byte for now.
 		bytes = bytes[n:]
-	} else {
+	} /*else {
 		// Old message format. No header present.
-	}
+	}*/
```

Comment:
Ahh, this looks a bit ugly to my eyes. Perhaps just do this:
```go
}
// If no header was present (the "if" check above), then this is the old message format. No header is present, so we don't do anything about the header.
```
Maybe you can phrase it a bit better. 
I just don't like the commented "else", it looks as if someone left half-done code that needs cleaning.